nixos/modules/services/logging/packetbeat.nix

```diff
@@ -5,6 +5,7 @@ with lib;
 let
   cfg = config.services.packetbeat;
   format = pkgs.formats.yaml {};
+  attrsToList = attrSet: map (name: {"type" = name;} // attrSet.${name}) (builtins.attrNames attrSet);
```

Comment:
@Infinisil This is the commit I talked about on IRC, however, this seems to only work on the default values, I have no idea how to make it apply to the settings set by a user.